### DISCUSSION TITLE
This should be below 5 seconds to minimize risk of losing rewards.

### DISCUSSION THREAD

**Discussion by chianew** [Chia-Network/chia-blockchain#4387] [Support]
2021-05-10T00:09:48.693 harvester chia.harvester.harvester: WARNING  Looking up qualities on J:\plot-k32-2021-04-29-15-14-fileid.plot took: 39.63419723510742. This should be below 5 seconds to minimize risk of losing rewards.
2021-05-10T00:12:01.281 harvester chia.harvester.harvester: WARNING  Looking up qualities on H:\plot-k33-2021-04-28-11-53-fileid.plot took: 5.224731683731079. This should be below 5 seconds to minimize risk of losing rewards.

Why did I get this log? How to fix it , thank you

**Comment by mumtazali2** [comment]
I also got few times. I guess at that time plotting was writing some plot to the disk which made it slow to respond. But not sure.

**Reply by chianew** [reply]
but it was 30 seconds to reach the reward, something changed?

**Reply by coderev3** [reply]
I was looking through logs and also see the same variation as well:

plot took: 12.143771648406982. This should be below 5 seconds to minimize risk of losing rewards.

**Reply by Keriwaza** [reply]
Same. Unsure.. I got that from a remote harvester's plots once or twice. I've since moved them to the main farmer. Like y'all I thought it was 30s to respond. Seems like this could use some clarification. Crowdsourced support will eventually get around to responding :) cheers

**Reply by 0xRapid** [reply]
https://chiaforum.com/t/new-warning-this-should-be-below-5-seconds-to-minimize-risk-of-losing-rewards/1285 they wrote about it there, it's 30 sec, 5sec is just a warning

**Comment by BrandtH22** [comment]
@chianew , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.

**Comment by Keriwaza** [comment]
Yep great support… five years later. Why even bother responding?On Dec 13, 2023, at 3:44 PM, ClydeWallace22 ***@***.***> wrote:﻿
Closed #4387 as resolved.
